config/initializers/permissions.rb

```diff
@@ -35,11 +35,17 @@
                      global: true,
                      contract_actions: { projects: %i[create] }
 
+      map.permission :archive_project,
+                     {
+                       'projects/archive': %i[create]
+                     },
+                     require: :member
```

Comment:
It might be a good idea to have `edit_project` as a dependency so that the user can reach the page (`projects/:id/settings/general`) where the button can be found. Of course, the user could achieve the same by visiting the project list but that is less intuitive. 

Probably more of a product question then it is a technical one.

Reply:
Indeed. I asked the question two weeks ago to the product team: https://community.openproject.org/projects/openproject/work_packages/3897/activity#activity-83. I'm still waiting for a reply.

Reply:
Parimal confirmed that it's ok if a user having only `archive_project` permission has to go to projects list to archive a project, and is not able to see the project settings page.
https://community.openproject.org/work_packages/3897/activity#activity-86